delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -560,20 +560,21 @@ def list_storage_host_groups(self, storage_id):
                         for host_info in host_info_list:
                             host_id = host_info.get('serial-number')
                             host_group_id = host_info.get('host-group')
-                            if host_group_id == storage_host_group_id:
-                                if hosts_str:
-                                    hosts_str = "{0},{1}".format(hosts_str,
-                                                                 host_id)
-                                else:
-                                    hosts_str = "{0}".format(host_id)
-                                storage_host_group_relation = {
-                                    'storage_id': storage_id,
-                                    'native_storage_host_group_id':
-                                        storage_host_group_id,
-                                    'native_storage_host_id': host_id
-                                }
-                                storage_host_grp_relation_list.\
-                                    append(storage_host_group_relation)
+                            if host_id != 'NOHOST':
+                                if host_group_id == storage_host_group_id:
+                                    if hosts_str:
```

Comment:
Please simplify the code, for example:
You can define 'hosts_str' as list type, last use ' ','join(hosts_str) '.
This also achieves the same result.

Reply:
The code has been optimized